server/src/MessageHandler.ts

```diff
@@ -131,6 +135,46 @@ function getBlockAtPosition(line: number, lines: Array<string>): Block | void {
   return
 }
 
+function getModelOrEnumBlock(
+  blockName: string,
+  lines: string[],
+): Block | void {
+  // get start position of model type
+  const results: number[] = lines
+    .map((line, index) => {
+      if (
+        (line.includes('model') && line.includes(blockName)) ||
```

Comment:
Are comment lines already excluded? Otherwise this would also be `true` for commented lines.

Reply:
Good point, I'll change that